pkg/apis/core/v1beta1/defaults.go

```diff
@@ -212,7 +140,7 @@ func SetDefaults_Shoot(obj *Shoot) {
 
 		if k8sVersionGreaterOrEqualThan122, _ := versionutils.CompareVersions(kubernetesVersion, ">=", "1.22"); !k8sVersionGreaterOrEqualThan122 {
 			// Error is ignored here because we cannot do anything meaningful with it.
-			// k8sVersionGreaterOrEqualThan122 will default to `false`.
+			// k8sVersionLessThan116 and k8sVersionGreaterOrEqualThan122 will default to `false`.
```

Comment:
From where `k8sVersionLessThan116` is coming?
```suggestion
			// k8sVersionGreaterOrEqualThan122 will default to `false`.
```